helix-term/src/commands.rs

```diff
@@ -1414,7 +1414,7 @@ fn copy_selection_on_line(cx: &mut Context, direction: Direction) {
         let (head, anchor) = if range.anchor < range.head {
             (range.head - 1, range.anchor)
```

Comment:
Do we need this here too?

Reply:
I don't think it's possible for `range.head` to be `0` in this branch because of the if condition: `range.anchor < range.head` implies `range.head` is `>= 1` unless it's a zero-width selection which should not be possible